### PR TITLE
Add QBTC Exchange To the Site

### PR DESCRIPTION
**Pull request by RoboticMind**
Adds an image for the exchange.
Adds an entry to the navbar, footer, exchange page, and _aquire_details.htm

### CODE REVIEW

**Comment by barton2526** [comment]
Blocking for a little bit, let's pump the brakes until we get more information

File: _includes/_acquire_details.htm [hunk 1]
```diff
@@ -77,6 +77,11 @@ <h3>Centralized exchanges</h3>
                             <img width="300" src="../assets/img/Exchanges/flypme.png" alt="Flyp.Me" />
                         </a>
                     </li>
+                    <li>
+                        <a href="https://www.myqbtc.com/trade?symbol=GRC_CNYT">
+                           <img width="300" src="../assets/img/Exchanges/qbtc.png" alt="qbtc">
```
Comment (barton2526) [inline _includes/_acquire_details.htm]: should the alt be capitalized?

File: exchange.htm [hunk 2]
```diff
@@ -57,6 +57,11 @@ <h3>Centralized exchanges</h3>
                             <img width="300" src="../assets/img/Exchanges/flypme.png" alt="Flyp.Me" />
                         </a>
                     </li>
+                    <li>
+                        <a href="https://www.myqbtc.com/trade?symbol=GRC_CNYT">
+                           <img width="300" src="../assets/img/Exchanges/qbtc.png" alt="qbtc">
```
Comment (barton2526) [inline exchange.htm]: same as above

File: _includes/_acquire_details.htm (hunk 1 above)
Comment (RoboticMind) [reply _includes/_acquire_details.htm]: The other exchanges `alt=` tags barring Flyp.me were not capitalized so I figured it should also probably not be. 🤷‍♂

**Comment by barton2526** [comment]
QBTC appears to be dead currently.  Closing PR tomorrow unless someone can justify keeping it.

**Comment by RoboticMind** [comment]
Think I'll just close it now

**Pull request closed without merging**